tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -28,6 +42,34 @@ pub struct Layer<
 
 /// A builder for [`Layer`](struct.Layer.html) that logs formatted representations of `tracing`
 /// events and spans.
+///
+/// ```rust
+/// use tracing_subscriber::{
+///     fmt::{self, format::Format, time},
+///     layer::Layer as _,
+///     registry::Registry,
+///     EnvFilter,
+/// };
+///
+/// let default_filter = concat!(module_path!(), "=info");
+///
+/// let fmt = Format::default().with_timer(time::Uptime::default());
+///
+/// let fmt_layer = fmt::Layer::builder()
+///     .event_format(fmt)
+///     .with_target(false)
+///     .finish();
+///
+/// let filter_layer = EnvFilter::try_from_default_env()
+///     .or_else(|_| EnvFilter::try_new(default_filter))
+///     .unwrap();
+///
+/// let subscriber = filter_layer
+///     .and_then(fmt_layer)
+///     .with_subscriber(Registry::default());
+///
+/// tracing::subscriber::set_global_default(subscriber).unwrap();
```

Comment:
on the fence about showing constructing & setting a subscriber in this example...

Reply:
I think if you want to keep the examples minimal you shouldn't, so in the examples in the fmt_layer.rs file they shouldn't be shown, but in the examples in fmt/mod.rs they should be shown.